package.json

```diff
@@ -43,6 +43,7 @@
   "devDependencies": {
     "chai": "3.5.0",
     "coffee-19-jshint": "2.3.1",
+    "coffee-jshint": "0.2.3",
```

Comment:
I agree. I think it was because we were stucked to coffeescript 1.9, but it isn't the case anymore. Let's wait for @aenario ;)